packages/terminal/src/browser/terminal-preferences.ts

```diff
@@ -97,12 +97,13 @@ export const TerminalConfigSchema: PreferenceSchema = {
             default: false
         },
         'terminal.integrated.cursorStyle': {
-            description: 'Controls the style of the terminal cursor/',
+            description: 'Controls the style of the terminal cursor.',
             enum: ['block', 'underline', 'line'],
             default: 'block'
         },
         'terminal.integrated.cursorWidth': {
             description: 'Controls the width of the cursor when \'cursorStyle\' is set to \'line\'.',
```

Comment:
nitpick, but while we are at it let's also fully qualify `cursorStyle` in the `description` field:
```ts
description: 'Controls the width of the cursor when \'terminal.integrated.cursorStyle\' is set to \'line\'.',
```

Reply:
Should be good to merge once this is done, I know nitpicks are annoying to address :)

Reply:
I have done the changes. Testing is annoying :-).